docs/v1/Makefile

```diff
@@ -1,21 +1,27 @@
-
 SPEC=P4Runtime-Spec
 
-all: html pdf build
+ROUGE_STYLE=github
+ROUGE_CSS=style
+
+all: ${SPEC}.pdf ${SPEC}.html
 
 build:
 	mkdir -p build
 
-# Note: Brute-force image rendering; could use more precise file-by-file make rules
-images: build assets/*.odg
```

Comment:
I think we need to restore lines like these in the Makefile, otherwise running `make` in the docs/v1 directory fails to find the .svg and .png image files.

Reply:
Indeed, we have 2 use cases:

- If the Docker runs on a host machine, the user needs to generate the PNG images and then compile.
- If the Docker has to run on a CI server, the images must be generated within the make process or checked in beforehand.

Reply:
But we can reduce all cases to behave in a way where whether it is a bare metal system, a VM, or a container, the Makefile has the commands that generate the .svg/.png files from the OpenOffice source files, by installing LibreOffice on the system, yes?  I thought that was the preference that Chris expressed earlier, and that is why we only have the OpenOffice files checked in, not the .svg/.png files.

Even if there are cases where the .svg/.png files are sitting there, running `make` with a Makefile that contains rules  for generating those files will only generate them if they are needed, so I don't see only advantages to having these rules in the Makefile that can generate them.

Reply:
Sorry, one more quick comment, in case this is something I am confused about.

If there are any containers anywhere that anyone wants to use to generate the PDF & HTML for the P4Runtime spec, I think we should tell them: "Install LibreOffice in that container, or it won't work."

Reply:
Totally agree with you. Does this mean we need to implement it for the container running in the CI environment?

Reply:
Agreed, good catch.